quote-book/synchronized/src/lib.rs

```diff
@@ -1,111 +1,392 @@
 // Copyright (c) 2023 MobileCoin Inc.
-
-use deqs_quote_book_api::{Error, Pair, Quote, QuoteBook, QuoteId};
+// Copyright (c) 2023 MobileCoin Inc.
```

Comment:
Why is this here?

Reply:
Sloppiness in the merge I did. Sorry about that.